README.md

```diff
@@ -91,6 +92,7 @@ You can also optionally specify:
 - [`--color-scheme`](#color-schemes) choose a different color scheme
 - [`--font-family`](#fonts) choose an OS installed font (default: monospace)
 - [`--font-size`](#fonts) specify font point size (default: 12)
+- [`--toolkit`](#toolkits) specify the toolkit (default: java2d)
 Image format determined by filename extension, supported extensions are `.png`, `.webp`, `.jpeg` and `.jpg`.
 - `--line-delay` if your play script runs a command like `lein repl`, the underlying program might not be ready to accept input immediately.\
```

Comment:
Probably a previous bad edit on my part, but:
> Image format determined by filename extension, supported extensions are `.png`, `.webp`, `.jpeg` and `.jpg`.

Belongs to `--out` option.

Reply:
yea, I can fix that.